src/utility/CMakeLists.txt

```diff
@@ -4,21 +4,19 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 set(header_path "${BSMPT_SOURCE_DIR}/include/BSMPT/utility")
-set(header
-    ${header_path}/utility.h ${header_path}/Logger.h
-    ${header_path}/asciiplotter.h ${header_path}/parser.h
-    ${header_path}/spline.h ${header_path}/const_velocity_spline.h)
-set(src utility.cpp Logger.cpp asciiplotter.cpp parser.cpp spline.cpp
-        const_velocity_spline.cpp)
+set(header ${header_path}/utility.h ${header_path}/Logger.h
+           ${header_path}/parser.h ${header_path}/const_velocity_spline.h)
+set(src utility.cpp Logger.cpp parser.cpp const_velocity_spline.cpp)
 add_library(Utility ${header} ${src})
 target_include_directories(Utility PUBLIC ${BSMPT_SOURCE_DIR}/include
                                           ${BSMPT_BINARY_DIR}/include)
 if(Boost_FOUND)
-  target_link_libraries(Utility PUBLIC Boost::boost)
+  target_link_libraries(Utility PUBLIC Boost::boost ASCIIPlotter Spline)
```

Comment:
```suggestion
  target_link_libraries(Utility PUBLIC Boost::boost)
```

Reply:
You can remove this here and a line `target_link_libraries(Utility PUBLIC ASCIIPlotter Spline)` outside the if condition of boost and nlohmann_json

Reply:
Done.

Reply:
I think the commit for this is missing